domain/src/main/kotlin/no/nav/su/se/bakover/domain/brev/LagBrevRequest.kt

```diff
@@ -35,6 +35,8 @@ interface LagBrevRequest {
                 satsGrunn = behandlingsinformasjon.getSatsgrunn().orNull()!!,
                 satsBeløp = beregning.getSats().månedsbeløpSomHeltall(beregning.periode.tilOgMed),
                 satsGjeldendeFraDato = beregning.getSats().datoForSisteEndringAvSats(beregning.periode.tilOgMed).ddMMyyyy(),
+                // Innvilgede vedtaker har alltid forventet inntekt
+                forventetInntektStørreEnn0 = behandlingsinformasjon.uførhet!!.forventetInntekt!! > 0,
```

Comment:
En spesiell grunn til at dette bare gjøres ved innvilgelse? Så vidt jeg kan se er dette noe som inkluderes via "brevkomponenten" for beregning og ville dermed antatt at dette var aktuelt i alle tilfeller hvor vi har en beregning?

Videre; kan dette være en anledning til å prøve å finne denne verdien fra data på nytt format (uføregrunnlag)?

Reply:
Jeg antar vi ikke har en perfekt løsning på dette for Vilkår enda heller? Et vilkår kan være IkkeVurdert, hvordan skal dette visualiseres? Samtidig kan det ha ingen vurderingsperioder. Men den kan brukes :) I praksis så burde vi hatt en ny type som garanterte for dette, når vi er Beregnet kunne vi hatt en NEL og fjernet muligheten for IkkeVurdert.

Reply:
oppgaven nevnte spesifikt innvilgede, men er ikke egentlig noe imot at alle brev med beregninger får den. 

kan se på å bruke uføregrunnlag ja

Reply:
@hestad Kunne nok gjort noen utbedringer i modellene i forhold til bruk av NEL, uavklart, mm. som kunne hjulpet oss på vei. Burde vel dog være mulig å lage en algoritme som finner ut av avslagsgrunnene (på samme måte som for behandlingsinformasjon) for de vilkårene som faktisk er vurdert (trenger vel bare visualisere den/de tingene som faktisk er vurdert til NEI på samme måte som i dag?) - skal vel godt gjøres at man havner her uten å ha vurdert noe som helst.